great-idea/css/index.css

```
@@ -63,4 +63,110 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 80%;
+	margin: auto;
+}
+
+header {
+	margin-top: 50px;
+}
+
+#topbar {
+	display: flex;
+	justify-content: space-between;
+}
+
+nav {
+	width: 800px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+nav a {
+	text-decoration: none;
+	color: lightgray;
+	font-size: 18px;
+}
+
+.logo {
+	height: 40px;
+}
+
+#banner {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 650px;
+	border-bottom: 2px black solid;
+}
+
+div.banner-content {
+	width: 600px;
+	text-align: center;
+}
+
+.banner-content h1 {
+	font-size: 100px;
+	margin-bottom: 30px;
+}
+
+.banner-content a {
```

You should be using a button tag instead of an a tag for "Get Started". Also, consider adding a hover state using. For example

```suggestion
.banner-content button:hover {
background-color: palevioletred;
color: white;
```